### PR TITLE
Refactor for-loop in getRegionStations

### PR DESCRIPTION
**Pull request by NullRa**

### CODE REVIEW

File: Common/MainMapViewModel.swift [hunk 1]
```diff
@@ -86,14 +86,14 @@ class MainMapViewModel {
         let minLng = center.longitude - region.span.longitudeDelta / 2
         var annotations: [MKPointAnnotation] = []
 
-        for index in 0 ..< ubikeDatas.count {
-            if ubikeDatas[index].longitude > minLng && ubikeDatas[index].longitude < maxLng
-                && ubikeDatas[index].latitude > minLat && ubikeDatas[index].latitude < maxLat {
+        for value in ubikeDatas {
```
Comment (ender503) [inline Common/MainMapViewModel.swift]: Give meaningful naming, e.g. station 😏

**Comment by ender503** [comment]
The commit title is longer than 50 characters, how about simplified into `Refactor for-loop in getRegionStations()`? 😏 
Alt text

**Comment by NullRa** [comment]
OK

**Comment by ender503** [comment]
Why the commit message contains `?` ? 🤔

**Comment by NullRa** [comment]
embarrassed

**Comment by ender503** [comment]
It's okay. Let's merge this branch! 🚀

**Pull request merged**